source/sonata_config.rst

```diff
@@ -203,6 +203,8 @@ There is also one additional field `type` used to denote the population type.
                                                   * :ref:`vasculature <vasculature_node_type>`
 
                                                Default is `biophysical`.
+   spatial_index_dir              Optional     Path to the directory containing the spatial index files.
```

Comment:
I think we should be explicit about the index type; in case we end up with more of them.  So, in this case, it would be the `spatial_segment_index_dir`

Reply:
if it is spatial index for segments, should it not be `segment_spatial_index_dir`?

Reply:
In spatial-index they are called "segment index" and "synapse index" so we could keep the same naming: `segment_index_dir` and `synapse_index_dir` (even though the executables are called `spatial-index-nodes` and `spatial-index-synapses`).

Adding a `spatial_` prefix as `spatial_segment_index_dir` and `spatial_synapse_index_dir` can work as well, but `segment_spatial_index_dir` sounds odd to me.

Reply:
Yeah, it's a trade-off between the natural sounding naming, and, IMO, `grouping` things with similar functionality together (so they sort well, etc).  I think of it as a namespace, without the complexity of a subdictionary.